Example/Podfile

```diff
@@ -1,6 +0,0 @@
-source 'https://github.com/CocoaPods/Specs.git'
-use_frameworks!
-
-target 'YelpAPI_Example', :exclusive => true do
```

Comment:
We could also share the YelpAPI_Example scheme, but I'm not totally clear on which schemes need to be shared and which don't. (The YelpAPI scheme does need to be shared since we test it.)